src/main/java/com/intel/gkl/compression/IntelDeflater.java

```diff
@@ -39,9 +40,14 @@
  * Provides a native Deflater implementation accelerated for the Intel Architecture.
  */
 public final class IntelDeflater extends Deflater implements NativeLibrary {
+    private final static Logger logger;
     private static final String NATIVE_LIBRARY_NAME = "gkl_compression";
     private static boolean initialized = false;
```

Comment:
Can add assignment for these too in dedicated static block

Reply:
Done !